service-api/module/Application/config/module.config.php

```diff
@@ -196,6 +198,46 @@
                     ],
                 ],
             ],
+            'find_postoffice_branches' => [
+                'type'    => Segment::class,
+                'options' => [
+                    'route'    => '/yoti/post-office-branches',
```

Comment:
I think it would be good to remove "yoti" from the URLs here, since it ties us a bit to the supplier. I think it's fine in file/class names but since the URLs form a contract they should describe what they do rather than how they do it.

Reply:
Yeah, if you want to keep it abstract` /counter-service/branches` is more generalised.